front-end/src/screens/Settings/address.tsx

```diff
@@ -183,74 +175,82 @@ const Address = ({ className }: Props): JSX.Element => {
 		);
 	}
 
-	if (accounts.length === 0) {
+	const addressList = (accounts: InjectedAccountWithMeta[], title: string, showOnlyUnlink: boolean, showAccounts: boolean) => {
```

Comment:
can we change these params to be an object? It'd help a lot to read when it's called.

Reply:
Done